src/content/chainlink-automation/overview/automation-economics.mdx

```diff
@@ -13,7 +13,7 @@ import { TabsContent } from "@components/Tabs"
 
 Chainlink Automation only requires an execution fee for transactions onchain. This fee includes the transaction cost, a node operator percentage fee (refer to the formula below), and a small fixed gas overhead accounting for gas between the network and the registry. The percentage fee compensates the Automation Network for monitoring and performing your upkeep. The Automation percentage fee varies by chain and is listed on our [Supported Networks](/chainlink-automation/overview/supported-networks) page.
 
-Fees for upkeeps that are configured for LINK payments have a conversion from native tokens to LINK, and this conversion is not performed for upkeeps configured for native token payments. Otherwise, the fee formula is the same:
+Fees for upkeeps that are configured for LINK payments (or other non-native payments as applicable) have a conversion from native tokens to LINK, and this conversion is not performed for upkeeps configured for native token payments. Otherwise, the fee formula is the same:
```

Comment:
Fees for upkeeps that are configured for LINK payments (or other non-native payments as applicable) have a conversion from native tokens to LINK (or other non-native payments as applicable), and this conversion is not performed for upkeeps configured for native token payments. Otherwise, the fee formula is the same: